src/app.js

```diff
@@ -17,6 +20,16 @@ const oAuthPort = process.env.PORT_OAUTH || port; // eslint-disable-line no-proc
 const configurationPort = process.env.PORT_CONFIG || port; // eslint-disable-line no-process-env
 const splitMode = oAuthPort !== configurationPort;
 
+if (config.datasources && config.datasources.postgres) {
+  const sequelize = new Sequelize(config.datasources.postgres.uri, {logging: log.info});
+  config.datasources.postgres.models = PostgresModels(sequelize, config.datasources.postgres.forceDBSync);
```

Comment:
I think it would be better if we had migrations, and no calls to sync(..) in the code at all. But that can probably wait until next iteration.